{
  "Header": {
    "Original": "CVE-2024-28835 (A flaw has been discovered in GnuTLS where an application crash can be ...)",
    "ID": "CVE-2024-28835",
    "Description": "(A flaw has been discovered in GnuTLS where an application crash can be ...)"
  },
  "Annotations": [
    {
      "Original": "- gnutls28 \u003cunfixed\u003e (bug #1067463)",
      "Type": "package",
      "Package": "gnutls28",
      "Kind": "unfixed",
      "BugNo": 1067463
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=2269084",
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=2269084"
    },
    {
      "Original": "NOTE: https://gitlab.com/gnutls/gnutls/-/issues/1525",
      "Type": "NOTE",
      "Description": "https://gitlab.com/gnutls/gnutls/-/issues/1525"
    },
    {
      "Original": "NOTE: https://gitlab.com/gnutls/gnutls/-/issues/1527",
      "Type": "NOTE",
      "Description": "https://gitlab.com/gnutls/gnutls/-/issues/1527"
    },
    {
      "Original": "NOTE: https://lists.gnupg.org/pipermail/gnutls-help/2024-March/004845.html",
      "Type": "NOTE",
      "Description": "https://lists.gnupg.org/pipermail/gnutls-help/2024-March/004845.html"
    },
    {
      "Original": "NOTE: https://www.gnutls.org/security-new.html#GNUTLS-SA-2024-01-23",
      "Type": "NOTE",
      "Description": "https://www.gnutls.org/security-new.html#GNUTLS-SA-2024-01-23"
    }
  ]
}